### ISSUE TITLE
preview images that are gamma & contrast corrected, maybe also cropped

### ISSUE THREAD

**Issue by krpandit** [nygctech/hudson#30]

**Comment by krpandit** [comment]
updated pyseq-image to use class methods to open different types of image from tiffs, xarray created zarr, ome-zarr created zarr, pyseq RoughFocus, and pyseq ObjStack

To use:
im = HiSeqImages.open_tiffs(PATH)
im = HiSeqImages.open_zarr(PATH) # will work for xarray of ome-zarr created zarr

Added tests for opening from tiffs, xarray created zarr, ome-zarr.

Also create new function to create preview images from 5D images that have channel and cycle dimensions or 4D images with a marker dimension, along with a test for the 5D case. 

TODO: 

- [ ] update snakemake files for opening images
- [ ] save preview images somewhere in pipeline

**Comment by krpandit** [comment]
updated get_HiSeqImages in pyseq-image to use classmethods to open tiffs or zarr so we don't have to update the snakemake files.

see commit 39bb50c in the pyseq-image preview branch

**Comment by krpandit** [comment]
I updated unmix, picasso, and pyseq-image, environments to use  pyseq-image updated with class methods to open files.

The picasso script was updated to use pyseq-image to open zarr file and old commented out code was deleted.
 
I found a bug in the segmentation script when using both a nuclei and cytoplasm image. The cytoplasm and nuclei channel index was [1,2] respectively but it was changed to [0,1]. 

The unmix script was updated to use the new pyseq-image class methods to open zarr file from processed zarr and then save ome-zarrs. Also added code to write preview images, and added a section specific preview directory to the output of the unmix snakemake rule. I also deleted old commented out code.
